indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -521,6 +521,31 @@ def _process(self):
                                                     user_data=self._get_user_data(), management=True)
 
 
+class RHRegistrationCreateMultiple(RHManageRegFormBase):
+    """Create multiple registrations for Indico users (management area)"""
+
+    def _register_user_personal_data_only(self, user, notify):
```

Comment:
Well, I could shorten it to `_register_user` and add a comment stating that it fills only the personal data fields.